themes/bootstrap3/templates/header.phtml

```diff
@@ -107,6 +108,10 @@
           <?php endif; ?>
         </ul>
       </nav>
+
+      <?php if(!($this->templateDir == 'search' && $this->templateName == 'home')): ?>
```

Comment:
Rather than hard-coding these values here, would it be better to have a flag in $this->layout() that could be set in search/home.phtml to bypass this notification? Then it would be possible to have custom notifications in other places as well without having to touch the header template. You can look at our existing use of `$this->layout()->breadcrumbs` as an example of the sort of thing I have in mind.

Reply:
I think I understood the flag and have committed the changes.